tests/test_config.yml

```diff
@@ -53,14 +48,6 @@ saga:
         - name: DeleteOrder
           controller: tests.services.SagaTestService.SagaService
           action: delete_order
-    queue:
```

Comment:
In this case, `saga.broker` and `saga.port` must be also removed as they are already defined at `broker.host` and `broker.port`